src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -3133,6 +3133,16 @@ var projectUniqueNamesForBuildIntToUpdate
                         log: logger,
                         token: token);
                 }
+
+                foreach (var parent in parents)
```

Comment:
Isn't this equivalent to what lines 3108-3117 are doing? 

What has changed?

Reply:
Yes, it's. Now parent is already evaluated by Line 3123-3134. We'll discuss one more time tomorrow.

Reply:
Addressed your concern now it's moved to 1 line in Line 3116.